### PR TITLE
Improved message for field not found

### PR DESCRIPTION
**Pull request by caarlos0**
I think that the current message is rather hard to understand,
  especially if the field that was not found is named "name"... I tried
  to improve the messages a bit, not sure if it is good though.

### CODE REVIEW

File: named.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func bindArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{
 	fields := m.TraversalsByName(v.Type(), names)
 	for i, t := range fields {
 		if len(t) == 0 {
-			return arglist, fmt.Errorf("could not find name %s in %#v", names[i], arg)
+			return arglist, fmt.Errorf("could not find the field named \"%s\" in \"%#v\"", names[i], arg)
```
Comment (elithrar) [inline named.go]: You can use `%q` here instead of quoting the verbs yourself, and/or use raw strings (backticks) to avoid having to escape the quotes.

Comment (caarlos0) [inline named.go]: Good catch, done.

Comment (caarlos0) [inline named.go]: Don't know how to do it with `%#v` though

Comment (elithrar) [inline named.go]: I would just use single quotes there - e.g. '%#v' - as %q also uses single
quotes.

Comment (caarlos0) [inline named.go]:
```go
package main

import "fmt"

func main() {
	fmt.Printf("blah %q", "asdsads")
}
```

```
$ go run main.go
blah "asdsads"
```

I think it's double quotes...

Comment (elithrar) [inline named.go]: Sorry, you're right. You probably don't need to quote the args list as %#v
is sufficiently obvious (being a syntax representation).

Comment (caarlos0) [inline named.go]: right, thanks!

**Comment by caarlos0** [comment]
@elithrar done

**Comment by caarlos0** [comment]
@elithrar ping

**Comment by caarlos0** [comment]
@elithrar ping

**Comment by robert-zaremba** [comment]
This PR is pretty obvious. What is blocking from merging it?

**Comment by caarlos0** [comment]
well, I give up.

**Comment by elithrar** [comment]
Sorry: I never saw the original pings and don't have merge rights here (I was reviewing as a fellow user).

**Pull request closed without merging**